packages/manager/modules/bm-server-components/src/interventions/translations/Messages_fr_FR.json

```
@@ -0,0 +1,6 @@
+{
+  "server_configuration_intervention_type": "Type",
+  "server_configuration_intervention_date": "Date",
+  "server_configuration_intervention_empty": "Aucune intervention",
+  "server_configuration_intervention_error": "Une erreur est survenue lors du chargement des informations"
```

May be we have to pass error details message ?

Like :
```suggestion
  "server_configuration_intervention_error": "Une erreur est survenue lors du chargement des informations : {{ message }}"
```

---

Good suggestion. This is legacy code. I have just moved/split it to new file and module. The aim of the refactoring is to do with minimal code changes. This can be taken as a separate task.